### DISCUSSION TITLE
Export result as vtu

### DISCUSSION THREAD

**Discussion by hbadi** [mfem/mfem#3103] [Q&A] [answered]
Hello,
I'm a new user of mfem, is there a way to export solutions as vtu file format in order to use paraview ?

Best regards.

**Comment by rcarson3** [comment, marked as answer]
@hbadi you can make use of the `ParaViewDataCollection` class to do this. A small example of how to use it is viewable in Example 9 here: https://github.com/mfem/mfem/blob/master/examples/ex9.cpp#L353-L365  and more in-depth documentation can found by searching for the class at https://mfem.org/dox/

**Reply by JianboLong** [reply]
@rcarson3 hello, do you know how we can visualize an iteratively refined mesh step by step in Paraview, rather the final mesh ? Or do you know where are such tutorials ? 

Bests
